release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -651,6 +651,15 @@ You must still use the `performance-addon-operator-must-gather` image when runni
 [id="ocp-4-11-insights-operator"]
 === Insights Operator
 
+==== Insights Operator data collection enhancements
+In OpenShift Container Platform 4.11, the Insights Operator collects the following additional information:
+
+* Cluster `images.config.openshift.io` resource definitions
+* Error messages from the `kube-controller-manager` container logs 
+* The `StorageCluster` resource from clusters with the `ocs-operator` or `odf-operator` installed
```

Comment:
Maybe I would say just `storageclustes.ocs.openshift.io/v1` resources

Reply:
so do you want me to cut off the part about the operators?

Reply:
I introduced the typo here...let me fix

Reply:
> so do you want me to cut off the part about the operators?

Yeah that was my thinking.....but probably not a big deal so up to you